forest-node/terraform/backend.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
+    bucket = "forest-test-state"
```

Comment:
This information should be mentioned in the README

Reply:
This is in the README